common/src/main/java/io/smallrye/config/utils/ConfigSourceUtil.java

```diff
@@ -44,4 +47,12 @@ private ConfigSourceUtil() {
         return properties.entrySet().stream().collect(Collectors.toMap(e -> String.valueOf(e.getKey()),
                 e -> String.valueOf(e.getValue())));
     }
+
+    public static Map<String, String> urlToMap(URL locationOfProperties) throws IOException {
+        try (InputStream in = locationOfProperties.openStream()) {
+            Properties p = new Properties();
+            p.load(in);
```

Comment:
This should explicitly wrap with an `InputStreamReader` for `StandardCharsets.UTF_8` otherwise it'll want Latin-1 input.

Reply:
Fixed, thanks!